packages/gitbook/src/middleware.ts

```diff
@@ -685,14 +686,23 @@ async function lookupSiteByAPI(
         `lookup content for url "${url.toString()}", with ${lookup.urls.length} alternatives`,
     );
 
+    // When the cookie is pulled from the cookie, set redirectOnError when calling getPublishedContentByUrl to allow
```

Comment:
Is this meant to be `When the token is pulled from the cookie,`

Reply:
Yep thanks